PlaceRouteHierFlow/placer/ILP_solver.cpp

```diff
@@ -94,13 +105,13 @@ double ILP_solver::GenerateValidSolution(design& mydesign, SeqPair& curr_sp, PnR
       {
         double sparserow[1] = {1};
         int colno[1] = {id * 4 + 1};
-        if (!add_constraintex(lp, 1, sparserow, colno, GE, 0)) printf("error\n");
+        if (!add_constraintex(lp, 1, sparserow, colno, GE, 0)) spdlog::error("error");
       }
       // y>=0
       {
         double sparserow[1] = {1};
         int colno[1] = {id * 4 + 2};
-        if (!add_constraintex(lp, 1, sparserow, colno, GE, 0)) printf("error\n");
+        if (!add_constraintex(lp, 1, sparserow, colno, GE, 0)) spdlog::error("error");
```

Comment:
I see some files using spdlog and some using cout. Are both being redirected consistently?

Reply:
Most cout are removed or replaced by spdlog. spdlog is turned off (https://github.com/ALIGN-analoglayout/ALIGN-public/blob/feature/simple_python_flow/PlaceRouteHierFlow/toplevel.cpp#L334). Now only 3 lines will be output to the terminal (https://github.com/ALIGN-analoglayout/ALIGN-public/blob/feature/simple_python_flow/PlaceRouteHierFlow/toplevel.cpp#L330)

Reply:
printings are removed. No lines will output to the terminal.